File: java/src/main/java/com/lambdaschool/dogs/GDPjavaApplication.java
package com.lambdaschool.dogs;


import com.lambdaschool.dogs.model.GDPList;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ApplicationContext;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.web.servlet.DispatcherServlet;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

@EnableScheduling
@EnableWebMvc
@SpringBootApplication
public class GDPjavaApplication
{

    public static GDPList gdpList;
    public static void main(String[] args)
    {
        gdpList = new GDPList();

        ApplicationContext ctx = SpringApplication.run(GDPjavaApplication.class, args);
        DispatcherServlet dispatcherServlet = (DispatcherServlet)ctx.getBean("dispatcherServlet");
        dispatcherServlet.setThrowExceptionIfNoHandlerFound(true);
    }
}

File: kotin/src/main/java/com/lambdaschool/gdp/GDPjavaApplication.kt
package com.lambdaschool.gdp


import com.lambdaschool.gdp.model.GDPList
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.ApplicationContext
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.web.servlet.DispatcherServlet
import org.springframework.web.servlet.config.annotation.EnableWebMvc

@EnableScheduling
@EnableWebMvc
@SpringBootApplication
object GDPjavaApplication {

    lateinit var gdpList: GDPList
    @JvmStatic
    fun main(args: Array<String>) {
        gdpList = GDPList()

        val ctx = SpringApplication.run(GDPjavaApplication::class.java, *args)
        val dispatcherServlet = ctx.getBean("dispatcherServlet") as DispatcherServlet
        dispatcherServlet.setThrowExceptionIfNoHandlerFound(true)
    }
}

